### ISSUE TITLE
Migrator still using https despite of V1_USE_HTTP=true specified

### ISSUE THREAD

**Issue by w1ndy** [docker/migrator#76]
Hi,

I'm migrating a v1 registry without SSL to a v2 registry with SSL. Both registries don't need authentication. Here is the command I ran:
```bash
docker run -it \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -e V1_REGISTRY=10.76.2.13:5002 \
    -e V2_REGISTRY=10.76.2.13:5001 \
    -e NO_LOGIN=true -e V1_USE_HTTP=true docker/migrator
```
and it fails with
```
FATA[0004] Error response from daemon: Get https://10.76.2.13:5002/v1/_ping: EOF
```

**Comment by w1ndy** [comment]
Quick fix: add `` --insecure-registry=10.76.2.13:5002`` to startup arguments of Docker daemon solved the problem. Perhaps this should be included in the docs?

Best Regards

**Comment by mbentley** [comment]
Correct, the daemon still needs to be configured to interact with an insecure registry. Good point about the docs, will update them when I am able to.
